ipi/inputs/forcefields.py

```diff
@@ -891,10 +891,10 @@ def store(self, ff):
                     _iobj = InputFFLennardJones()
                     _iobj.store(_obj)
                     self.extra[_ii] = ("fflj", _iobj)
-                elif isinstance(_obj, FFQUIP):
-                    _iobj = InputFFQUIP()
-                    _iobj.store(_obj)
-                    self.extra[_ii] = ("ffquip", _iobj)
+                # elif isinstance(_obj, FFQUIP):
```

Comment:
if this is gone I think it should be deleted. I agree I cannot find it anywhere else in i-PI and these lines were last touched in 2022. what I don't understand is why it is only now an error.